platform/ios/src/MGLMapViewDelegate.h

```diff
@@ -234,17 +234,20 @@ NS_ASSUME_NONNULL_BEGIN
  A value of `0.0` results in a completely transparent shape. A value of `1.0`,
  the default, results in a completely opaque shape.
  
+ @deprecated Use `-mapView:strokeColorForShapeAnnotation:`.
```

Comment:
Does jazzy understand the `@deprecated` keyword yet? If not, we could remove the documentation for this method.